applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -422,6 +436,21 @@ static void cell_pos_req_wk(struct k_work *work)
 	}
 }
 
+static void wifi_pos_req_wk(struct k_work *work)
+{
+	int err;
+
+	/* Request location using Wi-Fi access point info */
+	err = nrf_cloud_location_request(NULL, &wifi_data, true, NULL);
+	if (err) {
+		LOG_ERR("Failed to request Wi-Fi location, error: %d", err);
+		rsp_send("\r\n#XWIFIPOS: %d\r\n", err);
```

Comment:
What kind of error codes are replied to the AT-command? They are not listed in the documentation.